frontend/src/component/events/EventTimeline/EventTimelineEventGroup/EventTimelineEventGroup.tsx

```diff
@@ -31,9 +32,13 @@ export const EventTimelineEventGroup = ({
     const eventTime = group[0].timestamp;
 
     const position = `${((eventTime - startTime) / timelineDuration) * 100}%`;
+    const { trackEvent } = usePlausibleTracker();
+    const trackHover = () => {
+        trackEvent('event-timeline-event-hover');
+    };
 
     return (
-        <StyledEvent position={position}>
+        <StyledEvent position={position} onMouseOver={trackHover}>
```

Comment:
Have we validated that this only happens **once** when you hover the element? I assumed the correct event would be onMouseEnter 🤔

Reply:
I can try debugging it and see what happens.